### ISSUE TITLE
You should create a development branch aside for the default branch: Best Practice

### ISSUE THREAD

**Issue by jackie1santana** [Jxck-S/plane-notify#14]
I believe you should avoid releasing contributed code directly to the multi branch but instead create an additional branch called `multi-development`.

The Development branch would then be utilized in conjunction of best practices such as proof of concept, feature testing, vulnerability scanning, or bug fixes before release to production also known as the `main (multi branch)`. The main purpose of this branch is to avoid any potential code conflicts which could lead to inadvertent disfigurement of the code base.

_(upon approval of the contributors code tested, then merge the dev branch into production aka the main branch)_

I would like to contribute to the new branch as well. Pls notify if one is created. Tks.

**Comment by jwebbstevens** [comment]
@jackie1santana i agree. standard practice in open source projects. if this is done i can help with the wiki and other github pages and project structure.

**Comment by jackie1santana** [comment]
> @jackie1santana i agree. standard practice in open source projects. if this is done i can help with the wiki and other github pages and project structure.

totally, a wiki page for a legendary repo like this would be awesome. I would love to see that ! @jwebbstevens

**Comment by gabrielblancogarcia** [comment]
Absolutely agree @jackie1santana @jwebbstevens

**Comment by matkoniecz** [comment]
> The main purpose of this branch is to avoid any potential code conflicts which could lead to inadvertent disfigurement of the code base.

This can be entirely avoided without having dev branches.

**Comment by Jxck-S** [comment]
Thanks for the feedback. It’s been a while, and with more experience now I agree. I’ll be uploading a cleaned up and refactored version soon, and a development branch is definitely part of the plan.
